code/game/objects/items/weapons/tanks/tanks.dm

```diff
@@ -115,6 +115,19 @@
 
 		qdel(src)
 
+/obj/item/tank/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
```

Comment:
Maybe typecheck here given that a monkey could suicide with this.

Reply:
Would that be a terrible thing? It doesn't seem to runtime, if that's the concern. Otherwise, since it still sets the mob's brain to suiciding, they won't be able to rejoin the round after this anyway so I don't think it's too big of a deal.